packages/select/stories/SingleSelect.stories.tsx

```diff
@@ -0,0 +1,192 @@
+import type { StoryObj } from '@storybook/react';
+import type { Key } from 'react';
+
+import { Chip } from '@launchpad-ui/chip';
+import { Item, Section } from '@react-stately/collections';
+import { useState } from 'react';
+
+import { SingleSelect, SingleSelectTrigger } from '../src';
+import { FRUIT, SECTIONED_ITEMS } from '../src/__tests__/constants';
+import { CustomSingleSelectTrigger } from '../src/__tests__/examples';
+
+export default {
+  component: SingleSelect,
+  title: 'Components/Select/Single',
+  description:
+    'A Select combines a text input with a listbox, allowing users to filter a list of options to items matching a query.',
+  parameters: {
+    status: {
+      type: import.meta.env.PACKAGE_STATUS__SELECT,
+    },
+  },
+};
+
+type Story = StoryObj<typeof SingleSelect>;
+
+export const Basic: Story = {
+  render: () => {
+    const Component = () => {
+      return (
+        <SingleSelect
+          label="Fruit"
+          defaultItems={FRUIT}
```

Comment:
let's think more about the `defaultItems` vs `items` thing. personally i'd have a hard time keeping this straight